cmd/geth/dbcmd.go

```diff
@@ -86,20 +86,18 @@ Remove blockchain and state databases`,
 		},
 	}
 	dbInspectCmd = &cli.Command{
-		Action:    inspect,
-		Name:      "inspect",
-		ArgsUsage: "<prefix> <start>",
-		Flags: slices.Concat([]cli.Flag{
-			utils.SyncModeFlag,
-		}, utils.NetworkFlags, utils.DatabaseFlags),
+		Action:      inspect,
+		Name:        "inspect",
+		ArgsUsage:   "<prefix> <start>",
+		Flags:       utils.DatabaseFlags,
 		Usage:       "Inspect the storage size for each type of data in the database",
 		Description: `This commands iterates the entire database. If the optional 'prefix' and 'start' arguments are provided, then the iteration is limited to the given subset of data.`,
 	}
 	dbCheckStateContentCmd = &cli.Command{
 		Action:    checkStateContent,
 		Name:      "check-state-content",
 		ArgsUsage: "<start (optional)>",
-		Flags:     slices.Concat(utils.NetworkFlags, utils.DatabaseFlags),
+		Flags:     utils.DatabaseFlags,
```

Comment:
Network flags are used for datadir initialization. Checkout https://github.com/ethereum/go-ethereum/blob/master/cmd/geth/config.go#L158

Reply:
I think the network flags are only applied here https://github.com/ethereum/go-ethereum/blob/a775e68421595d9c3807e68cce7ff2037991a781/cmd/utils/flags.go#L1432-L1445

as for the dbcmd, the `--datadir` flag is required, so I think it's sufficient. 

Please correct me if I'm missing anything.

Reply:
`--datadir` is not mandatory. The default dir can still be used if nothing is specified